backend/app/models/visualizations.py

```diff
@@ -0,0 +1,46 @@
+from datetime import datetime
+from typing import Optional, List
+
+from beanie import Document, View, PydanticObjectId
+from pydantic import BaseModel, Field
+
+from app.models.files import FileContentType
+from app.models.users import UserOut
+
+
+class VisualizationsBase(BaseModel):
+    name: str = "N/A"
+    description: Optional[str] = None
+
+
+class VisualizationsIn(VisualizationsBase):
+    pass
+
+
+class VisualizationsPatch(BaseModel):
+    name: Optional[str]
+    description: Optional[str]
+
+
+class VisualizationsDB(View, VisualizationsBase):
+    creator: UserOut
+    created: datetime = Field(default_factory=datetime.utcnow)
+    modified: datetime = Field(default_factory=datetime.utcnow)
+    bytes: int = 0
+    content_type: FileContentType = FileContentType()
+
+    class Settings:
+        name = "Visualizations"
+
+
+class VisualizationsList(View, VisualizationsBase):
```

Comment:
I don't think we need this, unless we need a view to link visualization configs to permissions. In the case of datasets and files, we are using mongo views to include authorization information. Here we are not doing that. For example:

```
    class Settings:
        source = DatasetDB
        name = "datasets_view"
        pipeline = [
            {
                "$lookup": {
                    "from": "authorization",
                    "localField": "_id",
                    "foreignField": "dataset_id",
                    "as": "auth",
                }
            },
        ]
```

Reply:
agree. I removed this.